application/F3DStarter.cxx

```diff
@@ -6,15 +6,29 @@
 #include "F3DOptionsParser.h"
 #include "F3DSystemTools.h"
 
+#define DMON_IMPL
+#ifdef WIN32
+#pragma warning(push)
+#pragma warning(disable : 4505)
+#include "dmon.h"
+// dmon includes Windows.h which defines 'ERROR' and conflicts with log.h
```

Comment:
Can you try to put `#define WIN32_LEAN_AND_MEAN` before `#include "dmon.h"` to see if `ERROR` is still defined?

Reply:
does not work sadly.